From b385da232fb288a96e7f5ec669d2146b7d60d5b5 Mon Sep 17 00:00:00 2001
From: Pawel Moll <pawel.moll@st.com>
Date: Thu, 21 Aug 2008 10:20:30 +0100
Subject: [PATCH] sound_stm: Frequency warning message removed from fsynth code.

The "WARNING! fsynth.n should not generate frequency [...] (12.288MHz
is maximum)" message removed, as the frequency synthesizer functional
spec claims its capabilities up to 216MHz!

Signed-off-by: Pawel Moll <pawel.moll@st.com>
---
 sound/stm/fsynth.c |    7 -------
 1 files changed, 0 insertions(+), 7 deletions(-)

diff --git a/sound/stm/fsynth.c b/sound/stm/fsynth.c
index aba8fb8..1184800 100644
--- a/sound/stm/fsynth.c
+++ b/sound/stm/fsynth.c
@@ -293,13 +293,6 @@ static int snd_stm_fsynth_channel_configure(struct snd_stm_fsynth *fsynth,
 	snd_stm_assert(channel >= fsynth->channels_from, return -EINVAL);
 	snd_stm_assert(channel <= fsynth->channels_to, return -EINVAL);
 
-	/* All specs say, that maximum frequency generated by the fsynth
-	 * is 12.288MHz... */
-	if (frequency > 12288000)
-		snd_stm_printe("WARNING! %s should not generate frequency %d!"
-				" (12.288MHz is maximum)\n",
-				fsynth->bus_id, frequency);
-
 	/*             a
 	 * F = f + --------- * f = f + d
 	 *          1000000
-- 
1.5.3.6

